peach_tuntiwong/week02/day10/js/atm.js

```diff
@@ -0,0 +1,74 @@
+
+$(document).ready(function(){
+
+  window.checkingBalance = 0;
+  window.savingsBalance = 0;
+
+  var isChecking0Balance = function() {
+    if (checkingBalance === 0) {
+      $('#checking-balance').css('background-color', 'red');
+    } else if (checkingBalance !== 0) {
+      $('#checking-balance').css('background-color', '#E3E3E3');
+    }
+  }
+
+  var isSavings0Balance = function() {
+    if (savingsBalance === 0) {
+      $('#savings-balance').css('background-color', 'red');
+    } else {
+      $('#savings-balance').css('background-color', '#E3E3E3');
+    }
+  }
+
+  isChecking0Balance();
+  isSavings0Balance();
+
+  var checkingBalanceUpdate = function(amount) {
+    checkingBalance += amount;
+    $("#checking-balance").html('$'+checkingBalance);
+    isChecking0Balance();
+  }
+
+  var savingsBalanceUpdate = function(amount) {
+    savingsBalance += amount;
+    $("#savings-balance").html('$'+savingsBalance);
```

Comment:
Ideally, for Separation of Concerns, we'd like to move these jQuery lines that change the DOM out of the functions for banking logic.